service/src/main/java/uk/gov/hmcts/reform/fpl/Application.java

```diff
@@ -5,18 +5,23 @@
 import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.cloud.openfeign.EnableFeignClients;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.FilterType;
 import org.springframework.retry.annotation.EnableRetry;
 import org.springframework.scheduling.annotation.EnableAsync;
+import uk.gov.hmcts.reform.auth.checker.spring.AuthCheckerConfiguration;
+import uk.gov.hmcts.reform.ccd.document.am.feign.CaseDocumentClientApi;
 
 @SpringBootApplication
 @EnableFeignClients(basePackages = {
     "uk.gov.hmcts.reform.idam.client",
     "uk.gov.hmcts.reform.rd.client",
     "uk.gov.hmcts.reform.fnp.client",
     "uk.gov.hmcts.reform.calendar.client",
-    "uk.gov.hmcts.reform.aac.client"
-})
-@ComponentScan
+    "uk.gov.hmcts.reform.aac.client",
+    "uk.gov.hmcts.reform.ccd.document.am",
+    }, basePackageClasses = {CaseDocumentClientApi.class})
+@ComponentScan(basePackages = "uk.gov.hmcts.reform", excludeFilters = {
+    @ComponentScan.Filter(type = FilterType.ASSIGNABLE_TYPE, classes = { AuthCheckerConfiguration.class}) })
```

Comment:
Some further findings:
1) We don't need to declare enabling FeignClient for am-document here. _uk.gov.hmcts.reform.ccd.document.am.config.**CaseDocumentManagementClientAutoConfiguration**_ already declare to enable FeignClients for package "_uk.gov.hmcts.reform.ccd.document.am_"

2) After removing those stuff in point 1, there is no more beans conflicts. Also, I think we don't need _@ComponentScan_ in our Application.java, as it is already included in _@SpringBootApplication_ annotation

Reply:
Updates on point 2.
We can't remove the _@ComponentScan_. A lots of unit test failed because they cannot find a bean of type '_uk.gov.hmcts.reform.fpl.service.time.Time_'. But, it doesn't affect FPL at all while running as a Springboot application. 

I didn't go deep into it, but can investigate the root cause if needed.